packages/hooks/src/useCountdown.ts

```diff
@@ -46,3 +46,10 @@ export function useCountdown(to: number): Countdown | undefined {
 
   return countdown
 }
+
+export const useIsCountdownOver = (to: number) => {
+  const target = dayjs.unix(to)
+  const relative = dayjs()
+  const diff = useMemo(() => target.diff(relative, 'second'), [target, relative])
```

Comment:
`relative` is always a new instance once this hook is called so the `useMemo` here is not doing its job.

Btw, I think we can reuse the `useCountdown` hook?

Reply:
just need to check if all the numbers are 0

Reply:
Sure, I will merge it to useCountdown